packages/cli/src/oclif/commands/team/get.js

```diff
@@ -48,10 +60,11 @@ class TeamListCommand extends ZapierBaseCommand {
 TeamListCommand.flags = buildFlags({ opts: { format: true } });
 TeamListCommand.description = `Get team members involved with your integration.
 
-These users come in two levels:
+These users come in three levels:
 
   * \`admin\`, who can edit everything about the integration
-  * \`subscriber\`, who can't directly access the app, but will receive periodic email updates. These updates include quarterly health socores and more.
+  * \`subscriber\`, who can't directly access the app, but will receive periodic email updates. These updates include quarterly health scores and more.
+  * \`collaborator\`, who has read-only access for the app, but will receive periodic email updates. These updates include quarterly health scores and more.
```

Comment:
Same comment as [this](https://github.com/zapier/zapier-platform/pull/539#discussion_r857359686). Maybe  `collaborator` should go before `subscriber`?